azure_iot_edge/CHANGELOG.md

```diff
@@ -2,73 +2,136 @@
 
 ## 3.3.1 / 2023-05-26
 
-* [Fixed] Add DEFAULT_METRIC_LIMIT for OpenMetrics-based checks. See [#14527](https://github.com/DataDog/integrations-core/pull/14527).
+***Fixed***: 
+
+* Add DEFAULT_METRIC_LIMIT for OpenMetrics-based checks. See [#14527](https://github.com/DataDog/integrations-core/pull/14527).
+
 
 ## 3.3.0 / 2022-09-16 / Agent 7.40.0
 
-* [Added] Update HTTP config spec templates. See [#12890](https://github.com/DataDog/integrations-core/pull/12890).
+***Added***: 
+
+* Update HTTP config spec templates. See [#12890](https://github.com/DataDog/integrations-core/pull/12890).
+
 
 ## 3.2.0 / 2022-05-15 / Agent 7.37.0
 
-* [Added] Support dynamic bearer tokens (Bound Service Account Token Volume). See [#11915](https://github.com/DataDog/integrations-core/pull/11915).
+***Added***: 
+
+* Support dynamic bearer tokens (Bound Service Account Token Volume). See [#11915](https://github.com/DataDog/integrations-core/pull/11915).
+
 
 ## 3.1.0 / 2022-04-05 / Agent 7.36.0
 
-* [Added] Add metric_patterns options to filter all metric submission by a list of regexes. See [#11695](https://github.com/DataDog/integrations-core/pull/11695).
-* [Fixed] Remove outdated warning in the description for the `tls_ignore_warning` option. See [#11591](https://github.com/DataDog/integrations-core/pull/11591).
+***Added***: 
+
+* Add metric_patterns options to filter all metric submission by a list of regexes. See [#11695](https://github.com/DataDog/integrations-core/pull/11695).
+
+***Fixed***: 
+
+* Remove outdated warning in the description for the `tls_ignore_warning` option. See [#11591](https://github.com/DataDog/integrations-core/pull/11591).
+
 
 ## 3.0.0 / 2022-02-19 / Agent 7.35.0
 
-* [Added] Add `pyproject.toml` file. See [#11318](https://github.com/DataDog/integrations-core/pull/11318).
-* [Fixed] Fix namespace packaging on Python 2. See [#11532](https://github.com/DataDog/integrations-core/pull/11532).
-* [Changed] Add tls_protocols_allowed option documentation. See [#11251](https://github.com/DataDog/integrations-core/pull/11251).
+***Changed***: 
```

Comment:
Non-blocking, does order matter? All the other PR order with Added, Fixed, then Changed.